ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/MigrationsTest.java

```diff
@@ -176,6 +220,10 @@ private static SourceDescription describeSource(final String name) {
     return RestIntegrationTestUtil.makeKsqlRequest(REST_APP, sql);
   }
 
+  private static List<String> makeKsqlQuery(final String sql) {
+    return RestIntegrationTestUtil.makeWsRequest(REST_APP.getWsListener(), sql, Optional.empty(), Optional.empty(), Optional.empty());
```

Comment:
How come we're using the websocket endpoint? The regular query endpoint (see `RestIntegrationTestUtil #makeQueryRequest()`) is preferable, though in an ideal world we'd refactor RestIntegrationTestUtil to use the new HTTP/2 endpoints instead.

Reply:
`makeQueryRequest ` was a private method. It seems safe to change to public though, so that's updated in the latest commit